models/product/src/product/index.ts

```diff
@@ -5,11 +5,10 @@ import {
   GraphqlModelBuilder,
   RestModelBuilder,
 } from './builders';
+import * as constants from './constants';
 import * as modelPresets from './presets';
 
 export * from './types';
-export * as constants from './constants';
```

Comment:
We're still namespacing the constants export but in a different way (see Product export below).